### ISSUE TITLE
Poster at PAGE 2023: Mechanistic modelling of bioavailability and local immunogenicity after subcutaneous administration within the Open Systems Pharmacology framework

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSP-based-publications-and-content#493]
Moriah Pellowe, Gianluca Selvaggio, Marylore Chenel, Erik Sjögren

https://www.page-meeting.org/default.asp?abstract=10497
[Poster](https://www.page-meeting.org/pdf_assets/10497-abstract.pdf)

**Introduction**:

The subcutaneous (SC) route of administration is frequently used to deliver therapeutic proteins (TPs). However, the systemic bioavailability of TPs when given SC is often incomplete, which can significantly reduce the systemic exposure and therapeutic efficacy. Local proteolysis in lysosomes at the site of injection can mediate pre-systemic degradation. Similarly, local resident and, upon stimuli of the injection, migrating immune cells could expediate both first-pass degradation and a signal to immunogenetic response with injection dependent consequences for the systemic immunogenicity. These two elements were incorporated into a previously described mechanistic absorption and systemic immunogenicity PBPK-QSP model implemented and harmonized to the Open Systems Pharmacology (OSP) framework, to allow for more informed predictions of the pharmacokinetics and immunogenicity of TP following SC administration.

**Objectives**:

- Implement mechanisms of size dependent extravasation, according to the 2-pore formalism, in the mechanistic model framework for subcutaneous (SC) administration
- Introduce local endosomal uptake, degradation, endothelial trafficking, and FcRn salvaging according to the generic implementation in PK-Sim.
- Develop a QSP model for local SC immune cell activity and response upon injection based on preclinical observations.
- Integrate established local SC immunogenicity and absorption model with systemic pharmacokinetics and immunogenicity within the OSP framework.
- Evaluate implementations in terms of bioavailability and immunogenicity.  

**Methods**:

All activities were performed within MoBi and aligning to implementations in PK-Sim to allow for full integration in the Open Systems Pharmacology framework. Previously described models for local distribution and absorption after SC administration and systemic immunogenicity was used as starting point. Implementation of the generic distribution model of TPs in PK-Sim was incorporated in the SC absorption model to accommodate for local endosomal uptake, degradation, endothelial trafficking, and FcRn salvaging. Evaluation of this implementation was conducted within a range of molecular sizes and FcRn affinities of TPs reported in literature.

A model of the local immune response, inspired by a published work for lung infection, was developed. The model included different cell dynamics, e.g., recruitment, maturation, and migration to nearest lymph node, of key cellular players. Data from observations in Rhesus Macaques for specific class of immune cells (e.g., neutrophils, dendritic cells, monocytes and plasmacytoids) were used for model calibration. The final implementation was then integrated with the SC absorption model and linked to systemic immunogenicity for anti-drug antibody (ADA) response. Pre-systemic loss of drug after administration could consequently be influenced by both immune cell uptake and ADA mediated elimination.

**Results**:

Implemented models were used to predict absorption rate, bioavailability, and stimuli of local immunogenicity after SC administration based on a few molecular descriptors, e.g., size, FcRn affinity, epitope affinity. The model predicted size dependent retention and reduced systemic appearance rate with increased molecular size. First pass elimination via endosomal degradation increased because of increased retention while it was significantly reduced by FcRn salvaging. For a typical mAb (150 kDa) the predicted time to complete absorption was ~ 5 days, independent of FcRn salvaging,  and the bioavailability was >95% and <80% with (Kd<1 µM) and without FcRn salvaging, respectively. The model for local immune cell dynamics increased the systemic immunogenicity response dependent on molecular size and epitope potency with effects on both local and systemic PK after multiple dosing. This effect was mediated by local and downstream effects on immune cell differentiation and memory cell pool.

**Conclusion**:

In conclusion, our open-source platform for modelling SC administration in the Open Systems Pharmacology framework was further developed with mechanistic descriptions. The results demonstrated how the model platform could aid drug development through multi-layered analysis including drug absorption, bioavailability, and exposure as well as immunogenic response. This provides a valuable tool for predictions of exposure and efficacy of TP after SC administration including the possibility to assess immunogenicity consequences of switching from intravenous to SC route of administration.
